app/client/components/payment/update/CurrentPaymentDetail.tsx

```diff
@@ -0,0 +1,273 @@
+import { PaypalLogo } from "../paypalLogo";
+
+export function cardExpired(year: number, month: number) {
+  const expiryTimestamp = new Date(year, month);
+  const now = new Date();
+
+  return expiryTimestamp < now;
+}
+
+const CurrentPaymentDetails = (props: ProductDetail) => {
+  const { subscription } = props;
+
+  const mainPlan = getMainPlan(subscription);
+  const hasCancellationPending: boolean = subscription.cancelledAt;
+
+  const groupedProductType = GROUPED_PRODUCT_TYPES[props.mmaCategory];
+
+  const specificProductType = groupedProductType.mapGroupedToSpecific(props);
+
+  const keyValuePairCss = css`
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  `;
+
+  const keyCss = css`
+    ${textSans.medium({ fontWeight: "bold" })};
+    padding: 0 ${space[2]}px 0 0;
+    display: inline-block;
+    vertical-align: top;
+    width: 14ch;
+  `;
+
+  const valueCss = css`
+    ${textSans.medium()};
+    padding: 0;
+    display: inline-block;
+    vertical-align: top;
+    width: calc(100% - 15ch);
+
+    text-align: right;
+
+    ${minWidth.tablet} {
+      text-align: left;
+    }
+  `;
+
+  const hasPaymentFailure: boolean = !!props.alertText;
+
+  return (
+    <div
+      css={css`
+        border: 1px solid ${neutral[86]};
+        margin-bottom: ${space[6]}px;
+      `}
+    >
+      <div
+        css={css`
+          display: flex;
+          justify-content: space-between;
+          align-items: start;
+          background-color: ${hasCancellationPending
+            ? neutral[97]
+            : brand[400]};
+          ${minWidth.mobileLandscape} {
+            align-items: center;
+          }
+        `}
+      >
+        <h2
+          css={css`
+            font-size: 17px;
+            font-weight: bold;
+            margin: 0;
+            padding: ${space[3]}px;
+            color: ${hasCancellationPending ? neutral[7] : neutral[100]};
+            ${maxWidth.mobileLandscape} {
+              padding: ${space[3]}px;
+            }
+            ${minWidth.tablet} {
+              font-size: 20px;
+              padding: ${space[3]}px ${space[5]}px;
+            }
+          `}
+        >
+          {specificProductType.productTitle(mainPlan)}
+        </h2>
+        <div
+          css={css`
+            display: flex;
+            align-items: center;
+            padding: ${space[3]}px 0;
+            ${maxWidth.mobileLandscape} {
+              flex-direction: column;
+              align-items: end;
+            }
+          `}
+        ></div>
+      </div>
+      <div
+        css={css`
+          padding: ${space[5]}px ${space[3]}px;
+          ${minWidth.tablet} {
+            padding: ${space[5]}px;
+            display: flex;
+          }
+        `}
+      >
+        <div
+          css={css`
+            padding-bottom: ${space[3]}px;
+
+            ${minWidth.tablet} {
+              margin: ${space[6]}px 0 0 0;
+              padding: ${space[6]}px 0 0 0;
```

Comment:
these 2 lines get overwritten on lines 134 and 135

Reply:
thanks, there may be a couple of these in the PR